tests/vignette.js

```diff
@@ -480,7 +526,7 @@ QUnit.test('Thumbnailer recognizes vignette URLs vs static.wikia.nocookie.net/ U
 				height: 512
 			},
 			expectedOutput: 'https://static.wikia.nocookie.net/6f2431a7-19de-494c-91d0-391536b44377' +
-			'/scale-to-width/width/256'
+			'/scale-to-width/256'
```

Comment:
Yep, and it looks like travis on this repo doesn't mark the build as failure when the tests fail. I tried pushing without this to see if the fail was my env only and the test failed in Travis but the commit got a tick: https://travis-ci.org/Wikia/vignette-js/builds/185827503